### ISSUE TITLE
Accessibility: Notification Dialogue with toolbar violates  WCAG 4.1.2 Name , Role, Value  "SC 4.1.2 Name, Role, Value (Level A)"

### ISSUE THREAD

**Issue by RushikeshTrivedi** [microsoft/vscode#314777]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version:  1.119
- OS Version: Windows 11

Steps to Reproduce:
1. Any notification dialogue tool bar gives this issue

Issue#1 with Notification Dialogue:

Issue:
Multiple elements with "toolbar" roles do not have unique labels. 

For all user interface components (including but not limited to: form elements, links, and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies.

Rationale
This requirement ensures that Assistive Technologies (AT) can gather information about, activate, set, and receive status of all user interface components.

Note: This requirement is primarily for developers who develop or script their own custom user interface components. Standard HTML controls already meet this requirement when used according to specification.

Custom interactive user interface components often do not expose proper role, state, and property information needed for assistive technologies. As a result, people with disabilities have difficulty identifying and interacting with the components. Common examples are custom menus, trees, data grids, accordions, and dialog boxes. Ensuring proper markup and keyboard operation enables people with disabilities to identify and interact with the custom components in the same manner they identify and interact with standard components.

Requirement Link: https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

Screenshot for Dialogue:

<img width="3601" height="1254" alt="Image" src="https://github.com/user-attachments/assets/ed2311b6-6a49-4525-9108-3cee8ad76689" />

Issue#2 with Notification Dialogue:
The tabbable element's role 'list' is not a widget role

<img width="3743" height="1219" alt="Image" src="https://github.com/user-attachments/assets/e903ef67-7a0b-4fa0-8708-4ea70ae0e418" />

Issue#3 with Notification Dialogue:
The ARIA attributes "aria-setsize, aria-posinset, aria-selected" are not valid for the element <div> with ARIA role "dialog"

<img width="3734" height="1384" alt="Image" src="https://github.com/user-attachments/assets/ac2c74a3-386e-4126-b962-db8946aa99b7" />

**Comment by RushikeshTrivedi** [comment]
With version 1.126:

#1 is fixed

#2 not fixed:The tabbable element's role 'list' is not a widget role

<img width="3363" height="2102" alt="Image" src="https://github.com/user-attachments/assets/ff3ffa1a-89eb-4383-a8d8-3c6d37803f98" 
/>

#3: The ARIA attributes "aria-setsize, aria-posinset, aria-selected" are not valid for the element

with ARIA role "dialog"

<img width="3330" height="2105" alt="Image" src="https://github.com/user-attachments/assets/110dec6e-b0cc-41af-b3c5-14156fee8fe9" />
